bot/exts/events/advent_of_code/_cog.py

```diff
@@ -229,6 +229,10 @@ async def aoc_leaderboard(
         Additionally you can specify a `self_placement_name`
         that will append the specified profile's personal stats to the top of the leaderboard
         """
+        # Strip quotes from the self placement name if needed (e.g. "My Name" -> My Name)
+        if self_placement_name and self_placement_name[0] == self_placement_name[-1] == '"':
```

Comment:
What about using str.startswith/endswith? Might be more readable?

Reply:
I don't see how using that makes it more readable, I think it's fine as it is.

Reply:
Personally, I think the named methods are probably a bit easier to understand, especially with this project geared towards beginners.